cms/admin/pageadmin.py

```diff
@@ -1401,8 +1402,12 @@ def get_tree(self, request):
         """
         site = self.get_site(request)
         pages = self.get_queryset(request)
-        node_id = request.GET.get('nodeId')
-        open_nodes = list(map(int, request.GET.getlist('openNodes[]')))
+        node_id = re.sub(r'[^\d]', '', request.GET.get('nodeId', '')) or None
```

Comment:
Sorry, I am totally not a regex person. What does this do and why is it important?

Reply:
The (possibly naive) idea here is to remove all non-digit characters from the passed data. So this would remove all the possible values for [THOUSAND_SEPARATOR](https://docs.djangoproject.com/en/4.1/ref/settings/#std-setting-THOUSAND_SEPARATOR) that Django could pass in the request and just leave the raw integer value.